Bottom: 73adbc3c2f6c0a5dade41f877ecf8808416e583b
Top:    f5bbc06b9f55dd3d9006f5102cccd896e1d3b15d
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-16 15:13:21 -0500

Refresh of gitlab-10.8

---
diff --git a/roles/docker-gitlab/templates/before_backup.j2 b/roles/docker-gitlab/templates/before_backup.j2
index 6d882a32..74764935 100644
--- a/roles/docker-gitlab/templates/before_backup.j2
+++ b/roles/docker-gitlab/templates/before_backup.j2
@@ -3,6 +3,7 @@
 # *****************************************************************************
 # Backup gitlab
 docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z debian:latest bash -c "rm -rf /tmp/import_export/GIT"
+docker exec {{ gitlab_container_name }} /bin/bash -c "rm -rf {{ gitlab_bv_mountpoint}}/*"
 mkdir -p -m u=rwx,g=rwx,o= {{ gitlab_docker_backup_dir }}
 docker exec -t {{ gitlab_container_name }} gitlab-rake gitlab:backup:create
 docker exec -t {{ gitlab_container_name }} /bin/sh -c \
